mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -1,65 +1,43 @@
+void buildProject(String target, String cmakeOpts) {
+    cmakeBuild generator: 'Ninja',\
+        buildDir: 'build',\
+        buildType: 'RelWithDebInfo',\
+        installation: 'InSearchPath',\
+        steps: [[args: target]],\
+        cmakeArgs: "-DMLIR_MIOPEN_DRIVER_ENABLED=1 $cmakeOpts"
+}
+
+String dockerArgs() {
+    return '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+}
+
+String dockerImage() {
+    return 'rocm/mlir:rocm4.1-latest'
+}
+
 pipeline {
     agent {
         docker {
-            image 'rocm/mlir:rocm4.1-latest'
-            args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
-            label 'rocm'
+            image dockerImage()
+            args dockerArgs()
+            label "rocm && !single_gpu"
```

Comment:
Why disable single gpu?

Reply:
We probably don't need to for this - I'll fix this later this afternoon.

Reply:
Done